src/etcd_encoding.rs

```diff
@@ -4,7 +4,7 @@ use super::protobuf_gen::{
         api::{
             admissionregistration::v1::{MutatingWebhookConfiguration, ValidatingWebhookConfiguration},
             apps::v1::{DaemonSet, Deployment},
-            core::v1::{ConfigMap, Secret},
+            core::v1::{ConfigMap, Node, Secret},
```

Comment:
should not we change ip in node object too?

Reply:
I've been looking into it, looks like we can safely delete these annotations (which contain the IP):

```
k8s.ovn.org/host-cidrs
k8s.ovn.org/l3-gateway-config
k8s.ovn.org/node-chassis-id
k8s.ovn.org/node-mgmt-port-mac-address
k8s.ovn.org/node-primary-ifaddr
```

And they get recreated.

Same thing with the node status - if you delete it it gets recreated immediately

The only trouble I've had was with these:

```
machineconfiguration.openshift.io/currentConfig: rendered-master-482484efc39298d216b84f84e49ce1fb
machineconfiguration.openshift.io/desiredConfig: rendered-master-482484efc39298d216b84f84e49ce1fb
machineconfiguration.openshift.io/desiredDrain: uncordon-rendered-master-482484efc39298d216b84f84e49ce1fb
machineconfiguration.openshift.io/lastAppliedDrain: uncordon-rendered-master-482484efc39298d216b84f84e49ce1fb
machineconfiguration.openshift.io/lastSyncedControllerConfigResourceVersion: "278801"
```

Which contain the old MC rendered hashes. If you delete them, they don't get recreated, MCO doesn't if we do this. So we should leave them as is and just see what happens

Reply:
Both spot on comments, I was thinking of testing these first and then see what exactly needs to be done. I'll keep you posted, please let me know if you manage to verify any of these.